WorkloadManagementSystem/Executor/InputData.py

```diff
@@ -203,6 +224,12 @@ def _resolveInputData(self, jobState, inputData):
     if not result['OK']:
       self.jobLog.error("Failed to check replicas", result['Message'])
       return result
+    okReplicas = result['Value']
+
+    result = self.__getSiteCandidates(okReplicas, vo)
+    if not result['OK']:
+      self.jobLog.error("Failed to check SiteCandidates", result['Message'])
+      return result
```

Comment:
Why did you move this check here? (it was in `__checkReplicas`)

Reply:
because I reuse the call `__checkReplicas` for the Sandbox, and there we do not care about the site